ui/src/plugins/dev.perfetto.Tracks/index.ts

```diff
@@ -12,26 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import {Time} from '../../base/time';
 import {createAggregationToTabAdaptor} from '../../components/aggregation_adapter';
 import {
   metricsFromTableOrSubquery,
   QueryFlamegraph,
 } from '../../components/query_flamegraph';
+import {MinimapRow} from '../../public/minimap';
 import {PerfettoPlugin} from '../../public/plugin';
 import {AreaSelection, areaSelectionsEqual} from '../../public/selection';
 import {Trace} from '../../public/trace';
 import {SLICE_TRACK_KIND} from '../../public/track_kinds';
+import {LONG, NUM} from '../../trace_processor/query_result';
 import {Flamegraph} from '../../widgets/flamegraph';
 import {CounterSelectionAggregator} from './counter_selection_aggregator';
 import {PivotTableTab} from './pivot_table_tab';
 import {SliceSelectionAggregator} from './slice_selection_aggregator';
 
-/**
- * This plugin adds the generic aggregations for slice tracks and counter
- * tracks.
- */
 export default class implements PerfettoPlugin {
-  static readonly id = 'dev.perfetto.GenericAggregations';
+  static readonly id = 'dev.perfetto.Tracks';
+  static description =
+    'Generic bulk operations for tracks originating from the tracks table.';
```

Comment:
Why is this not in the TraceProcessorTrack plugin in that case? That's basically the responsibility of that plugin as well?

Reply:
Good idea, done!